Wireframe/index.html

```diff
@@ -1,13 +1,51 @@
 <!DOCTYPE html>
 <html lang="en">
-  <head>
+<head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Wireframe</title>
-    <link rel="stylesheet" href="style.css">
-  </head>
-  <body>
-    <!-- Add your HTML markup here -->
-    <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
+    <title>Git Essentials</title>
+    <link rel="stylesheet" href="css/style.css">
+</head>
+<body>
+    <!-- Page Header -->
+    <header>
+        <h1>Understanding Git</h1>
+        <p>The essential tool for modern software development</p>
+    </header>
+
+    <!-- Articles Section -->
+    <main>
+        <section class="articles">
```

Comment:
What is section doing in this code? What is the purpose? Can you talk through it?